### ISSUE TITLE
Surefire VM dies on Windows without apparent reason

### ISSUE THREAD

**Issue by reckart** [dkpro/dkpro-core#1097]
This happens after the unit tests in the corenlp module have completed and while Jenkins is recording test results.

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project de.tudarmstadt.ukp.dkpro.core.corenlp-gpl: Execution default-test of goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test failed: The forked VM terminated without properly saying goodbye. VM crash or System.exit called?
```

**Comment by reckart** [comment]
Might be related to

* https://issues.apache.org/jira/browse/SUREFIRE-1400
* https://issues.apache.org/jira/browse/SUREFIRE-1376

**Comment by reckart** [comment]
Probably won't work (upgrading to 2.20) since the issues listed above are marked as fixed only for 2.20.1 which is not released yet.

**Comment by reckart** [comment]
Upgrading surefire did help :)
